packages/toolpad-components/src/Container.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { Container as MUIContainer } from '@mui/material';
+import { createComponent } from '@mui/toolpad-core';
+
+interface Props {
+  children: React.ReactNode;
+  visible: boolean;
+}
+
+function Container({ children, visible, ...props }: Props) {
+  return visible ? (
+    <MUIContainer disableGutters sx={{ padding: 1 }} {...props}>
+      {children}
+    </MUIContainer>
+  ) : null;
+}
+
+export default createComponent(Container, {
+  argTypes: {
+    children: {
```

Comment:
Another thing: there is a bug I've found so if this prop is not called `children` it doesn't work, but it should. I'm including the fix in https://github.com/mui/mui-toolpad/pull/1527